### PR TITLE
Update EXPORT privileges

### LINKED ISSUE #9556: importccl: add more fine grained EXPORT privileges
PR: https://github.com/cockroachdb/cockroach/pull/57380

From release notes:
> [`EXPORT`](../v21.1/export.html) can now be used by non-`admin` users with `SELECT` privilege on the table being exported, unless the destination URI requires `admin` privileges. [#57380][#57380] {% comment %}doc{% endcomment %}

### PR DESCRIPTION
**Pull request by lnhsingh**
Closes #9556

### CODE REVIEW

File: v21.1/export.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ After the export has been initiated, you can cancel it with [`CANCEL QUERY`](can
 
 ## Required privileges
 
-Only members of the `admin` role can run `EXPORT`. By default, the `root` user belongs to the `admin` role.
+<span class="version-tag">New in v21.1:</span> The user must have the `SELECT` [privilege](authorization.html#assign-privileges) on the table being exported, unless the destination URI requires `admin` privileges.
```
Comment (adityamaru) [inline v21.1/export.md:27]: Should we link to - https://www.cockroachlabs.com/docs/stable/import.html#source-privileges?

**Review by adityamaru — APPROVED** [review]

File: v21.1/export.md (hunk 1 above)
Comment (adityamaru) [reply v21.1/export.md:27]: Just to give users the ability to check which URIs require admin priveleges.

**Review by lnhsingh — COMMENTED** [review]
TFTR!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9847#-:-MUUB5Li28m5JOVGB0WO:b-814g8k)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @adityamaru)

___
*[v21.1/export.md, line 27 at r1](https://reviewable.io/reviews/cockroachdb/docs/9847#-MUAkKKNtiXKIxUFN9Kl-r1-27:-MUUB32h5HKpYIb244l0:btgld4o) ([raw file](https://github.com/cockroachdb/docs/blob/cb7b3223f815c027348deddea810840f85c7059d/v21.1/export.md#L27)):*
<details><summary><i>Previously, adityamaru (Aditya Maru) wrote…</i></summary><blockquote>

Just to give users the ability to check which URIs require admin priveleges.
</blockquote></details>

Makes sense to me! Added.

**Review by rmloveland — APPROVED** [review]
LGTM

**Pull request merged**